cmake/finders/FindXkbcommon.cmake

```diff
@@ -0,0 +1,105 @@
+#[=======================================================================[.rst
+FindXkbcommon
+-------------
+
+FindModule for xkbcommon and associated libraries
+
+.. versionchanged:: 3.0
+  Updated FindModule to CMake standards
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+This module defines the :prop_tgt:`IMPORTED` target ``xkbcommon::xkbcommon``.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module sets the following variables:
+
+``Xkbcommon_FOUND``
+  True, if all required components and the core library were found.
+``Xkbcommon_VERSION``
+  Detected version of found xkbcommon libraries.
+
+Cache variables
+^^^^^^^^^^^^^^^
+
+The following cache variables may also be set:
+
+``Xkbcommon_LIBRARY``
+  Path to the library component of xkbcommon.
+``Xkbcommon_INCLUDE_DIR``
+  Directory containing ``xkbcommon.h``.
+
+#]=======================================================================]
+
+# cmake-format: off
+# cmake-lint: disable=C0103
+# cmake-lint: disable=C0301
+# cmake-format: on
+
+include(FindPackageHandleStandardArgs)
+
+find_package(PkgConfig QUIET)
+if(PKG_CONFIG_FOUND)
+  pkg_search_module(PC_xkbcommon QUIET xkbcommon)
```

Comment:
Missing capitalisation, should be `PC_Xkbcommon`

Reply:
Seems resolved in the latest force push.